### DISCUSSION TITLE
Missing "Show generation progress in window title" setting

### DISCUSSION THREAD

**Discussion by sebaxakerhtc** [vladmandic/sdnext#735] [Q&A] [answered]
Hi to all!
Today I moved to this nice fork but cannot find important thing from original project in "User interface" settings was "Show generation progress in window title"
It's very helpful when you start some long-time generation process and switching to another tab, you still can see progress in window title.

**Comment by tkiikt** [comment]
I thought so too at first. However, it is more convenient to hear a sound when the generation is completed rather than watching the progress ;-P

check this: Settings -> Live previews, Play a sound when images are finished generating.

**Reply by sebaxakerhtc** [reply]
I saw it, but don't have sound device and what if sound will play when the user is afk?
Old way is simpler - one look at tab and you know if it was ended

**Reply by vladmandic** [reply, marked as answer]
I can set busy in title and clear it once it's done, that's much less frequent update than actual progress.

**Reply by sebaxakerhtc** [reply]
> I can set busy in title and clear it once it's done, that's much less frequent update than actual progress.

Yes - it'll be perfect!
